src/crimson/os/seastore/cache.cc

```diff
@@ -193,8 +193,12 @@ std::optional<record_t> Cache::try_construct_record(Transaction &t)
 {
   // First, validate read set
   for (auto &i: t.read_set) {
-    if (i->state == CachedExtent::extent_state_t::INVALID)
+    if (i->state == CachedExtent::extent_state_t::INVALID) {
+      logger().debug("{}, invalid cachedextent in read_set, {}",
+		     __func__,
+		     *i);
```

Comment:
i'd suggest move this change into its own commit. but i am inclined to drop it. as it happens all the time in our seastore implementation.

@athanatos what do you think?

Reply:
+1 to moving into a separate commit.  I'm fine with adding the debugging if it was useful.